packages/vats/src/core/startWalletFactory.js

```diff
@@ -18,7 +18,7 @@ const trace = makeTracer('StartWF');
 /**
  * @param {ERef<ZoeService>} zoe
  * @param {Installation<
- *   import('@agoric/smart-wallet/src/walletFactory').start
+ *   import('@agoric/smart-wallet/src/walletFactory').prepare
```

Comment:
this change can go away now, right, @turadg ?

Reply:
yes, I missed that one